.circleci/config.yml

```diff
@@ -1353,24 +1353,23 @@ workflows:
           requires:
             - hold
       - make-release:
-          <<: *release-tags
+          requires:
+            - hold
       - push-revenuecat-pod:
           requires:
             - make-release
-          <<: *release-tags
```

Comment:
from here on it's pretty much the same thing, just removing the tag filter since we're already filtering at the `when` level